src/transport/raw/MessageHeader.h

```diff
@@ -327,6 +343,21 @@ class PayloadHeader
         return *this;
     }
 
+    /** Set the Initiator flag bit. */
+    PayloadHeader & SetInitiator(bool inInitiator)
```

Comment:
Should we add serialization for the above flags?

I imagine if we set anything inside a 'Payload Header' it should be serialized over the wire. If it is not serialized/deserialized, it should not be part of the header class but rather transmitted separaterly.

Reply:
Yes, this field need to be serialized as the I flag in the exchange header.
I Flag (1 bit, position 0)
A single bit field indicating that the message was sent by the initiator of the exchange.

There are still some other changes we need to do in MessageHeader to align with the spec. Those changes is tracked by https://github.com/project-chip/connectedhomeip/issues/3306